src/OpenLoco/src/Windows/News/Ticker.cpp

```diff
@@ -220,14 +207,13 @@ namespace OpenLoco::Ui::Windows::NewsWindow::Ticker
             if (!newsStringChar)
                 break;
         }
-
+        strcat(buffer - 1, "This is some longer text to force more new lines even when not needed");
```

Comment:
This is debug code to ensure that the message spans >4 lines to check the >4 line code works.